packages/react-styled-core/src/Checkbox/index.js

```diff
@@ -0,0 +1,98 @@
+import React, { forwardRef } from 'react';
+import Box from '../Box';
+import ControlBox from '../ControlBox';
+import Icon from '../Icon';
+import useColorMode from '../useColorMode';
+import VisuallyHidden from '../VisuallyHidden';
+import checkboxStyles from './styles';
+
+const Checkbox = forwardRef(
+  (
+    {
+      id,
+      name,
+      value,
+      'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
+
+      defaultIsChecked,
+      checked,
+      disabled,
+      readOnly,
+      isIndeterminate,
```

Comment:
How about changing `isIndeterminate` to `indeterminate`?

Reply:
https://ant.design/components/checkbox/#Props
https://material-ui.com/api/checkbox/#props

Reply:
Done!